### PR TITLE
SPIN-1619: Making the Fast Property Value field an expanding text area.

### PR DESCRIPTION
**Pull request by zanthrash**
- Was just a text input which made adding/editing multiline values difficult.

@anotherchrisberry PTAL

### CODE REVIEW

File: app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html [hunk 1]
```diff
@@ -44,13 +44,14 @@
...
-        <input type="text"
-               name="value"
-               class="form-control input-sm "
-               data-purpose="fastProperty-value"
-               ng-model="newFP.property.value"
-               placeholder="Enter a fast property value"
-               required/>
+        <textarea
+              style="resize: vertical;"
+              type="text"
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html]: I don't think `type` is needed on a `textarea`.

Nice on the `resize: vertical` - I didn't know about that.

**Comment by anotherchrisberry** [comment]
LGTM - defer to you on the `type` attribute

File: app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html (hunk 1 above)
Comment (zanthrash) [inline app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html]: You are absolutely right.  Thanks for catching that.

Comment (anotherchrisberry) [inline app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html]: You are absolutely welcome.

**Pull request merged**